### PR TITLE
[DOC] better explanation of adduct mass file

### PR DESCRIPTION
**Pull request by cbielow**

### CODE REVIEW

File: src/openms/include/OpenMS/ANALYSIS/ID/AccurateMassSearchEngine.h [hunk 1]
```diff
@@ -235,7 +235,16 @@ namespace OpenMS
     only the absolute value is used since many FeatureFinders will only report positive charges even in negative ion mode.
     Entities with charge=0 are treated as "unknown charge" and are tested with all potential adducts and subsequently matched against the database.
 
-    A list of potential adducts can be given for each mode separately.
+    A file with a list of potential adducts can be given for each mode separately. 
+    Each line contains a chemical formula (plus quantor) and a charge (separated by semicolon), e.g.
+    M+H;1+ 
+    The M can be preceded by a quantor (e.g.2M, 3M), implicitly assumed as 1.
+    The chemical formula can contain multiple segments, separated by + or - operators, e.g. M+H-H20.
```
Comment (timosachsenberg) [inline src/openms/include/OpenMS/ANALYSIS/ID/AccurateMassSearchEngine.h]: H20 -> H2O

File: src/openms/include/OpenMS/ANALYSIS/ID/AccurateMassSearchEngine.h [hunk 2]
```diff
@@ -235,7 +235,16 @@ namespace OpenMS
...
+    Brackets are implicit per segment, i.e. M+H-H20 is parsed as M + (H) - (H20).
+    Each segment can also be preceded by a quantor, e.g. M+H-2H20 would parse as
+    M + (H) - 2x(H20).
+    If debug mode is enabled, the masses of each segment are printed for verification. In particular,
+    typing H20 (twenty H) is different from H2O (water).
```
Comment (timosachsenberg) [inline src/openms/include/OpenMS/ANALYSIS/ID/AccurateMassSearchEngine.h]: ok doesn't seem to be a bug - sorry. Still confusing to choose this example ;)

**Comment by cbielow** [comment]
right. I fixed it.

**Comment by timosachsenberg** [comment]
+1 on pass

**Comment by hroest** [comment]
i restarted the build

**Comment by cbielow** [comment]
all tests pass (1 clang timed out, again)

**Pull request merged**